operator/v1/types_network.go

```diff
@@ -321,6 +321,10 @@ type OVNKubernetesConfig struct {
 	// not using OVN.
 	// +optional
 	HybridOverlayConfig *HybridOverlayConfig `json:"hybridOverlayConfig,omitempty"`
+	// EnableIPsec enables IPsec encryption on all pod-pod traffic within the cluster. If unset, the plugin will default
+	// EnableIPsec to false.
+	// +optional
+	EnableIPsec *bool `json:"enableIPsec,omitempty"`
```

Comment:
Any particular reason why you need this bool pointer, if you still default that to false?

Reply:
Probably not. What would be the difference with EnableIPsec bool?

Reply:
I would rather drop the pointer **and** omitempty. This makes the feature discoverable easily.